<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomConfict</class>
 <widget class="QDialog" name="DicomConfict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #525F7B;
border-top-left-radius: 8px;
border-top-right-radius: 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>15</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>80</y>
     <width>816</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255,255,255,0.85);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>40</y>
      <width>451</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>导入患者的ID相同，但是有些属性不同，请手动选择</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>180</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>生日：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSex">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>200</width>
      <height>37</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>230</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBirthday">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>200</width>
      <height>37</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>130</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>200</width>
      <height>37</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBtnCancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>480</y>
     <width>120</width>
     <height>48</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #525F7B;
border-radius: 8px 8px 8px 8px;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnOK2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>480</y>
     <width>120</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #525F7B;
border-radius: 8px 8px 8px 8px;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #363D4A;
border-radius: 8px 8px 8px 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>widget</zorder>
  <zorder>pBtnCancel</zorder>
  <zorder>pBtnOK2</zorder>
  <zorder>lblBg</zorder>
  <zorder>lblTitle</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
